aws-mssql.yml

```diff
@@ -47,6 +72,26 @@ provision:
     type: string
     details: AWS secret key
     default: ${config("aws.secret_access_key")}
+  - field_name: aws_vpc_id
+    type: string
+    details: VPC ID for instance
+    default: ""
+  - field_name: instance_class
+    required: true
+    type: string
+    details: |
+      AWS DB instance class. Accepted values will depend on the selected engine and mssql_version.
+      See: https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/CHAP_SQLServer.html#SQLServer.Concepts.General.InstanceClasses
+    default: ""
+  - field_name: rds_subnet_group
+    type: string
+    details: AWS RDS subnet group already in existence to use
+    default: ""
+  - field_name: rds_vpc_security_group_ids
+    type: string
+    details: Comma delimited list of security group ID's for instance
+    default: ""
+    prohibit_update: true
```

Comment:
Just flagging this, we do the same for other offerings, but I am a bit unsure why we restrict updates here as I haven't seen this causing recreation.

Reply:
I prefer handling this as a separate PR.

https://www.pivotaltracker.com/story/show/185286312